hw/ip/otp_ctrl/data/otp_ctrl_img.h.tpl

```diff
@@ -0,0 +1,149 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+${fileheader}
+// AUTOGENERATED. Do not edit this file by hand.
+// See the hw/ip/otp_ctrl/data README for details.
+
+#ifndef OPENTITAN_HW_IP_OTP_CTRL_DATA_OTP_CTRL_IMG_H_
+#define OPENTITAN_HW_IP_OTP_CTRL_DATA_OTP_CTRL_IMG_H_
```

Comment:
Does the include guard need to be customized based on the input filename?

Based on the rest of this file, it looks like only one of these is ever intended to be `#included` into a program (ie: the type definitions of enums and structs).  If we ever had a use case where multiple otp image definitions needed to be included into a single program, this include guard would prevent that (and, of course, the type definitions would need to be moved to somewhere independent to avoid re-defining them).

Reply:
This is a good idea. I updated the include guards and moved the type definitions into a standalone library.